activemq-karaf/pom.xml

```diff
@@ -42,12 +42,12 @@
     <dependency>
       <groupId>org.ops4j.pax.logging</groupId>
       <artifactId>pax-logging-api</artifactId>
-      <version>1.11.4</version>
+      <version>1.11.14</version>
```

Comment:
Should probably add a property for this, perhaps dependencyManagement, to keep it consistent (I see from the dependency:trree there are a few different pax-logging versions in use in different modules).

Reply:
Agree, both activemq-karaf and karaf itest use pax-logging.